### DISCUSSION TITLE
low pause gc of Graal?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1069] [Ideas]
Hi:

For gaming and other client side apps
gc pause could be a factor affects user experiences
so low pause gc could help us solve this problem
For normal java code e.g. without native image & aot compiling zgc would be a perfect solution for that
https://wiki.openjdk.java.net/display/zgc/Main
but too sad is when it comes to aot & native image provided by GraalVM
currently there is no low pause gc. The only solution is g1 only available for Graal Enterprise version which is not open soured and not freee
so we have to use serial gc for android & ios app
Good news is 1 of low pause gc which is shenendoah gc starts to merged in
https://github.com/oracle/graal/issues/3472
So is it possible to enable this low pause gc e.g. zgc or shenendoah in native image or image building phase of maven plugin?

Thanks

**Comment by AlmasB** [comment]
I haven't yet explored GC-level fine-tuning, so I don't know much about how to configure them. I would suggest that you ask the Graal team.
